libraries/Microsoft.Bot.Builder.AI.LUIS/LuisApplication.cs

```diff
@@ -15,8 +15,8 @@ public class LuisApplication
         /// </summary>
         /// <param name="applicationId">LUIS _application ID.</param>
         /// <param name="endpointKey">LUIS subscription or endpoint key.</param>
-        /// <param name="azureRegion">Azure region with endpoint.</param>
-        public LuisApplication(string applicationId, string endpointKey, string azureRegion)
+        /// <param name="endpoint">LUIS endpoint to use.</param>
+        public LuisApplication(string applicationId, string endpointKey, string endpoint)
```

Comment:
endpointKey is not being argument / null checked checked.

Reply:
It is null whitespace and URI checked.  Not sure what more you could do...